vsintegration/src/FSharp.Editor/BraceMatchingService.fs

```diff
@@ -95,13 +95,16 @@ type internal FSharpBraceMatchingService() =
         member this.FindBracesAsync(document: Document, position: int, cancellationToken: CancellationToken): Task<Nullable<BraceMatchingResult>> =
             document.GetTextAsync(cancellationToken).ContinueWith(
                 fun (sourceTextTask: Task<SourceText>) ->
-                    try match getBraceMatchingResult(sourceTextTask.Result, Some(document.Name), [], position, cancellationToken) with
-                        | None -> Nullable()
-                        | Some(braceMatchingResult) -> Nullable(braceMatchingResult)
-                    with ex -> 
-                        Assert.Exception(ex)
-                        reraise()
-                , TaskContinuationOptions.OnlyOnRanToCompletion)
+                    if sourceTextTask.Status = TaskStatus.RanToCompletion then
+                        try match getBraceMatchingResult(sourceTextTask.Result, Some(document.Name), [], position, cancellationToken) with
```

Comment:
A bit surprised that F# doesn't already  have an Option -> Nullable helper method.